kong/pdk/request.lua

```diff
@@ -714,7 +714,9 @@ local function new(self)
   --
   -- @function kong.request.get_raw_body
   -- @phases rewrite, access, response, admin_api
-  -- @treturn string The plain request body.
+  -- @treturn string|nil The plain request body or nil if it does not fit into
+  -- the NGINX temporary buffer.
+  -- @treturn string|nil An error message.
```

Comment:
```suggestion
  -- @treturn nil|string An error message.
```
The order seems to be incorrect

Reply:
Agree, I also think the first type should be `nil`. I used the documentation of the `_REQUEST.get_body` function as reference and the order is `string|nil` maybe we should also update the documentation for this function.

https://github.com/Kong/kong/blob/073d2fd54c8049f87a60680a2ee4c3a7ca88b21a/kong/pdk/request.lua#L787-L793